tests/grain_size_strain_pinned.prm

```diff
@@ -0,0 +1,59 @@
+# A test for the grain size material model. The test imposes simple shear
+# across a 100 km square domain with a rheology derived from experiments on
+# olivine. The model domain is isobaric and initially isothermal.
+# Over time the model maintains nearly constant temperatures, pressures and
+# strain rates. The grain size in the test approaches the equilibrium
+# grain size for t -> infinity. At 4 Myr the grain size is
+# 1.46095252e-03 m, (TODO: test: which is close to the equilibrium value?).
+# Viscosity: 1.42574e18 Pa s.
+
+set Dimension                              = 2
+
+include $ASPECT_SOURCE_DIR/tests/grain_size_strain.prm
+
+subsection Material model
+  set Model name = grain size
+
+  subsection Grain size model
+    set Reference density                = 3400
+    set Thermal conductivity             = 0
+    set Thermal expansion coefficient    = 0
+    set Reference compressibility        = 0
+    set Viscosity                        = 1e18
+    set Minimum viscosity                = 1e16
+    set Reference temperature            = 1600
+    set Recrystallized grain size        =
+
+    set Grain growth activation energy       = 4e5
+    set Grain growth activation volume       = 0.0
+    set Grain growth rate constant           = 1.92E-010
+    set Grain growth exponent                = 3
+    set Average specific grain boundary energy = 1.0
+    set Work fraction for boundary area change = 0.1
```

Comment:
This parameter is not used anymore. Instead, you can set both the Minimum grain size reduction work fraction and the Maximum grain size reduction work fraction to 0.1.